docs/client-side/angular/sample/src/Models/IdentityModels.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using Microsoft.AspNet.Identity;
+using Microsoft.AspNet.Identity.EntityFramework;
+using Microsoft.Data.Entity;
+using Microsoft.Data.Entity.Metadata;
+using Microsoft.Framework.OptionsModel;
+
+namespace AngularSample.Models
+{
+    // Add profile data for application users by adding properties to the ApplicationUser class
+    public class ApplicationUser : IdentityUser
+    {
+
+    }
+
+    public class ApplicationDbContext : IdentityDbContext<ApplicationUser>
+    {
+        private static bool _created = false;
+        
+        public ApplicationDbContext()
+        {            
+            // Create the database and schema if it doesn't exist
+            // This is a temporary workaround to create database until Entity Framework database migrations 
```

Comment:
This code is still required for deployment since we don't have a proper deployment story yet. This is the same code we include in the project template. @rustd we should probably update the comment to say "applying migrations during deployment isn't supported" rather than saying migrations in general aren't supported.